src/resources/Pipelines/Pipelines.ts

```diff
@@ -1,10 +1,18 @@
 import API from '../../APICore';
 import Resource from '../Resource';
 import MLAssociations from './MLAssociations/MLAssociations';
-import {PipelineBackendVersion} from './PipelinesInterfaces';
+import {PipelineBackendVersion, PipelineModel} from './PipelinesInterfaces';
 
 export default class Pipelines extends Resource {
-    static baseUrl = '/rest/search/v2/admin/pipelines';
+    static getBaseUrl = (version?: 1 | 2) => {

```

Comment:
2 calls instead of 1 with version as param ?
I think it will be easier to remove it later